react-props/my-app/src/ButtonContainer.tsx

```diff
@@ -0,0 +1,11 @@
+export function ButtonContainer({ button }: Props) {
+  return (
+    <div>
+      <button>{button}</button>
+    </div>
+  );
+}
+
+type Props = {
```

Comment:
It's best to put the Props type right above the function where it's used. This will make more sense when you start putting multiple components in the same file.